app/views/pages/faqs.html.erb

```diff
@@ -6,6 +6,270 @@
   <div class="usa-width-three-fourths usa-layout-docs-main_content">
     <h1><%= @page_title %></h1>
 
-    <%# TODO Add page content %>
+    <%# TODO Hide the checkboxes when the screen is too narrow %>
+    <div class="usa-grid" id="faq-filters">
+
+      <div class="usa-width-one-third">
+        <fieldset class="usa-fieldset-inputs usa-sans">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="claims-checkbox" type="checkbox" value="claims">
+              <label for="claims">Claims</label>
```

Comment:
The `for` attribute's value should match the `<input>`'s `id` that the label is associated with. In this case, `<label for="claims-checkbox">`.